tests/features/ecas-login.feature

```diff
@@ -42,16 +42,47 @@ Feature: Login through OE Authentication
     And I should not see the link "Log out"
     And I should see the link "Log in"
 
+  Scenario: A user's information should update every login
+    # Login with an EULogin user.
+    Given I am on the homepage
+    And I click "Log in"
+    And I fill in "Username or e-mail address" with "Lisbeth.SALANDER@ext.ec.europa.eu"
+    And I fill in "Password" with "dragon_tattoo"
+    And I press the "Login!" button
+    And I click "Edit"
+    Then the "First Name" field should contain "Lisbeth"
+
+    # Edit the details.
+    When I fill in "First Name" with "New name"
+    And I press "Save"
+    Then I should see "The changes have been saved."
+
+    When I click "Edit"
+    Then the "First Name" field should contain "New name"
+
+    # Logout
+    When I click "Log out"
+    And I press the "Log me out" button
+    Then I should be on the homepage
+
+    # Login again and check the changed details.
+    When I click "Log in"
+    And I fill in "Username or e-mail address" with "Lisbeth.SALANDER@ext.ec.europa.eu"
+    And I fill in "Password" with "dragon_tattoo"
+    And I press the "Login!" button
+    And I click "Edit"
+    Then the "First Name" field should contain "Lisbeth"
+
+    #Logout to continue scenarios
+    When I click "Log out"
+    And I press the "Log me out" button
+    Then I should be on the homepage
+
   Scenario: A blocked user should not be able to log in
-    Given users:
-      | name      |
-      | lsalander |
-    And the user "lsalander" is blocked
-    When I am on the homepage
-    Then I should see the link "Log in"
-    And I should not see the link "Log out"
+    Given the user "lsalander" is blocked
```

Comment:
This might not work any more, the user `lsalander` doesn't exist at this point in the scenario so can't be blocked.

It is usually a good idea not to reuse user names etc from previous tests. Since these features can become very long with a large number of scenarios it can be difficult to figure out which exact test fails if they all use the same test data.

Reply:
The problem is that we can't use any user since they have to come from the EULogin instance. Currently the only two users available are the ones we are already using. I think finding a solution for this issue is out of the scope for this ticket but I acknoledge the problem you presented so I added extra steps to make sure the user is available.

Reply:
I cannot access this EULogin image, but can't we create test users in it? Having to rely on hardcoded user names seems pretty limiting, especially for testing purposes. We definitely need to have the ability to run tests against dozens of users, and users with special characters in their names etc. Anyway this is out of scope.